.github/workflows/update-gh-pages.yaml

```diff
@@ -0,0 +1,26 @@
+name: Update GitHub pages
+
```

Comment:
Maybe here would be good to add the `run-name` and have the GitHub tag in it.

Reply:
good idea, however GitHub already shows the tag that it's being run on, e.g.
<img width="725" alt="GitHub Actions screenshot" src="https://github.com/bakdata/kpops/assets/4771462/0484f2ae-5308-43a6-a59a-d86713a78de4">

Reply:
WDYT now?